ios/FluentUI/Core/ControlHostingView.swift

```diff
@@ -99,7 +95,15 @@ open class ControlHostingView: UIView {
         )
     }
 
-    private let hostingController: FluentUIHostingController = .init(rootView: AnyView(EmptyView()))
+    private let hostingController: UIHostingController = {
+        if #unavailable(iOS 16) {
```

Comment:
Why 16, if we aren't API gated?

Reply:
The API we wanted to replace our swizzling with is only available in 16.2, but we don't need the entire class in 16. We do still need it in iOS 14 and 15, though. So I assume there are some changes to `UIHostingController` only in 16 that fix what we have work arounds for.

Reply:
Ohh that's wild, I didn't realize that. Can you lightly document that as a comment inline?

Reply:
wait, if it is 16.2 shouldn't it be 16.2 explicitly? not #unavailable (ios 16)?

Reply:
The new API we wanted was in 16.2, but the fix for the bugs we were working around is in just 16. Tested on a 16.0 simulator.

Reply:
Also, I was saying the wrong version for when the APIs were added. They were added in 16.4.